pages/company/company.module.scss

```diff
@@ -0,0 +1,187 @@
+main {
```

Comment:
I think you copied the whole code form some other `.scss` file

remove the scss code which is not required for company page

Reply:
Is this resolved?

Reply:
yes